.ci/jobs/apm-integration-test-axis-pipeline-jjbb.yml

```diff
@@ -0,0 +1,46 @@
+---
+- job:
+    name: apm-integration-test-axis-pipeline-mbp
```

Comment:
the name of the job doesn't match the name of the file, aka missing the `-mbp`, not sure whether tha's the standard but just in case

Reply:
Oh, my bad, it was using the name before the rebase/reset I did to maintain the old ones. Fixing it asap